recipes/libsemigroups-python-bindings/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   # branch instead
   #git_url: {{ repo }}.git
   #git_tag: python-bindings
-  url: {{ repo }}/archive/python-bindings.tar.gz 
+  url: {{ repo }}/archive/conda-test.tar.gz 
```

Comment:
I can try that but I don't think that's the issue. Although this recipe depends on libsemigroups being installed (via conda), it seems that when the continuous integration tries to build the python bindings it cannot find the headers to include.